deployment/run.sh

```diff
@@ -24,6 +24,19 @@ es_ops_node_quorum=${ES_OPS_NODE_QUORUM:-$((es_ops_cluster_size/2+1))}
 es_ops_recover_after_nodes=${ES_OPS_RECOVER_AFTER_NODES:-$((es_ops_cluster_size-1))}
 es_ops_recover_expected_nodes=${ES_OPS_RECOVER_EXPECTED_NODES:-$es_ops_cluster_size}
 es_ops_recover_after_time=${ES_OPS_RECOVER_AFTER_TIME:-5m}
+# node selectors
+es_ns_key=$(echo $ES_NODESELECTOR | cut -f1 -d=)
+es_ns_value=$(echo $ES_NODESELECTOR | cut -f2 -d=)
```

Comment:
I think we should have a function here for turning the env var specification into a nodeselector. This could provide some validation and string safety as well as enable multi-label selectors (even though there's no way to enter one in the template, that could get fixed, or the env var could be directly edited instead of filled by parameter). Something like:

```
function join { local IFS="$1"; shift; echo "$*"; }
# untested, probably needs some fixes:
function extract_nodeselector() {
  local inputstring="${1//"/}"  # remove any errant double quotes in the inputs
  local -a selectors=()
  for keyvalstr in ${inputstring//\,/ }; do
    keyval=( "${keyvalstr//=/ }" )
    if [[ -n "${keyval[0]}" && -n "${keyval[1]}" ]]; then
      selectors=( "${selectors[@]}" "\"${keyval[0]}\": \"${keyval[1]}\"")
    else
      echo "Could not make a node selector label from '$keyval'"
      exit 255
    fi
    if [[ "${#selectors[*]}" > 0 ]]; then
      echo "nodeSelector: { $(join , ${selectors[*]}) }"
    fi
}
```

And then in your code:

```
es_ns=$(extract_nodeselector $ES_NODESELECTOR)
if [[ -n "${ES_NODESELECTOR}" ]]; then
	sed "/serviceAccountName/ i\
          $es_ns" templates/es.yaml | oc process ...
```